lib/puppet/provider/oneview_san_manager/c7000.rb

```diff
@@ -75,4 +55,12 @@ def parse_provider_uri
     return if @data['providerUri'].to_s[0..6].include?('/rest/')
     @data['providerDisplayName'] = @data.delete('providerUri')
   end
+
+  def resource_name
+    'SANManager'
+  end
+
+  def self.resource_name
```

Comment:
This is not being touched by tests. 
Should add something like:
```ruby
  it 'should be able to run through self instances' do
    expect(instance).to be
  end
```
to unit/integration